src/main/java/it/gov/pagopa/exception/RestExceptionHandler.java

```diff
@@ -26,14 +26,12 @@ public ResponseEntity<Object> handleBadRequest(final DataIntegrityViolationExcep
         return handleExceptionInternal(ex, bodyOfResponse, new HttpHeaders(), HttpStatus.BAD_REQUEST, request);
     }
 
-    @Override
     public ResponseEntity<Object> handleHttpMessageNotReadable(final HttpMessageNotReadableException ex, final HttpHeaders headers, final HttpStatus status, final WebRequest request) {
         final String bodyOfResponse = "Http Message Not Readable";
         // ex.getCause() instanceof JsonMappingException, JsonParseException // for additional information later on
         return handleExceptionInternal(ex, bodyOfResponse, headers, HttpStatus.BAD_REQUEST, request);
     }
 
-    @Override
```

Comment:
have you removed this Override because it will override nothing?
if yes, this is an error! this override was performed to override default behavior to handle MethodArgumentNotValidException  exception, this way you are suppressing it